docker/Dockerfile_commcarehq_base

```diff
@@ -42,3 +44,5 @@ RUN pip install \
     ipython \
     --user --upgrade \
     && rm -rf /root/.cache/pip
+
+EXPOSE 8000
```

Comment:
Yes, to get the ports mapped you need to add `--service-ports` to docker-compose run command. I've added this to `./dockerhq.sh bash`.